recipes/google-cloud-iam/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "google-cloud-iam" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bcfd1b6f934349b30af3673889d3908bfdf8b8ab05e16b8e80d2ca4003b3d13f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - enum34  # [ py < 34 ]
+    - google-api-core-grpc >=1.6.0
+    - pip
+    - python
+  run:
+    - enum34   # [ py < 34 ]
+    - google-api-core-grpc >=1.6.0
+    - python
+
+test:
+  imports:
+    - google
+    - google.cloud
+    - google.cloud.iam_credentials_v1
+    - google.cloud.iam_credentials_v1.gapic
+    - google.cloud.iam_credentials_v1.gapic.transports
+    - google.cloud.iam_credentials_v1.proto
+
+about:
+  home: https://github.com/GoogleCloudPlatform/google-cloud-python
+  license: Apache Software
```

We generally prefer the SPDX identifiers:
```suggestion
  license: Apache-2.0
```